File: Codeforces/round740/A.py
n = int(input())
x=[]
for i in range(0,n):
    r=1
    l=int(input())
    x =list(map(int, input().split()))
    y = sorted(x)
    while(x!=y):
        for k in range(0,len(x)-1):
            #print(k)
            if(x[k]>x[k+1]):
                temp=x[k]
                x[k]=x[k+1]
                x[k+1]=temp
                #print(x)
        r+=1
    if(r==1):
        r-=1
    print(r)


          

File: Codeforces/round740/A.cpp
#include <iostream>
#include <vector>

using namespace std;



int main(){
    int n , l;

    cin >> n;
    for(int i =0 ; i < n ; i++){
        cin >> l;
        vector<int> x(l);
        for(int j=0; j < l;j++){
            cin >> x[j];
        }
        for(int j = 0 ; j <l ;j++){
            int temp;
            for(int k =0 ; k < x.size()-1 ; k++){
            if(x[k]>x[i+1]){
                    temp = x[k];
                    x[k]=x[k+1];
                    x[k+1]=temp;
                }
            }
            for(int j =0; j < x.size();j++){
            cout << x[j]<<" ";
        }
            cout<<":";
        }
        for(int j =0; j < x.size();j++){
            cout << x[j]<<" ";
        }
        cout <<"\n";
    }

}
